README.md

```diff
@@ -161,15 +165,31 @@ For assigning input and secure input to playbook variables you can use the tags:
 
 Shadow will supply you with the Vault password, which is needed to decrypt some credentials to access the NUC.
 
+
+### Hand and Arm ###
+Hand and Arm is one of the main use-cases of the server_and_nuc playbook. It requires more variables to be passed in then just using hand_e on its own. The main variables that need to be set are related to the arms themselves, mainly the variables ethercat_right/left_arm variable and arm_ip_right/left. But also the product type needs to be defined as product=arm_hand_e.
+
+Example:
+
+```bash
+bash <(curl -Ls bit.ly/run-aurora) server_and_nuc_deploy --branch v2.1.5 --inventory production --read-secure customer_key reinstall=true bimanual=true product="arm_hand_e" image="public.ecr.aws/shadowrobot/dexterous-hand" tag="noetic-release" arm_ip_right="10.8.1.1" arm_ip_left="10.8.2.1" ethercat_right_arm="eno1" ethercat_left_arm="enx000ec6bfe175"
+```
+
+### Hand and Glove ###
+Hand and Glove is another main product of the server_and_nuc playbook. This also takes in a new product type with product=glove_hand_e now, it also has extra parameters relating to the glove that you are using for the product. These variables are found in the options above, but the glove specific variables are glove, real_glove, biotacs and polhemus_type. Another thing to denote about this product is that it's not based on the dexterous-hand image, but instead the shadow-dexterous-hand-glove image.
```

Comment:
Where is that options above? Can you rewrite this section as its not very clear?